### PR TITLE
[ews-build.webkit.org] Update checkout based on specific remote

### PR DESCRIPTION
**Pull request by JonWBedard**
#### a38f826ee3ae1e34dc111dd3427ab87a16d355c3
<pre>
[ews-build.webkit.org] Update checkout based on specific remote
<a href="https://bugs.webkit.org/show_bug.cgi?id=242130">https://bugs.webkit.org/show_bug.cgi?id=242130</a>
&lt;rdar://problem/96162066&gt;

Reviewed by Aakash Jain.

* Tools/CISupport/ews-build/steps.py:
(ConfigureBuild.add_pr_details): Set &apos;remote&apos; property.
(FetchBranches.run): Fetch &apos;origin&apos; and target remote, if applicable.
(UpdateWorkingDirectory):
(UpdateWorkingDirectory.run): Use raw &apos;git&apos; commands instead of &apos;update-webkit&apos;.
(UpdateWorkingDirectory.evaluateCommand): Deleted.
(CleanGitRepo.run): Remove &apos;fetch&apos; command.
(PushCommitToWebKitRepo.evaluateCommand): Add &apos;fetch&apos; step.
* Tools/CISupport/ews-build/steps_unittest.py:

Canonical link: <a href="https://commits.webkit.org/252364@main">https://commits.webkit.org/252364@main</a>
</pre>

### CODE REVIEW

**Comment by JonWBedard** [comment]
#2295 does the rename

**Comment by JonWBedard** [comment]
#2301 moves around the `ShowIdentifier` step.

File: Tools/CISupport/ews-build/steps.py [hunk 1]
```diff
@@ -574,7 +574,14 @@ def add_pr_details(self):
         owners = self.getProperty('owners', [])
         revision = self.getProperty('github.head.sha')
 
-        if self.getProperty('project') != 'WebKit/WebKit':
+        project = self.getProperty('project')
+        if project == GITHUB_PROJECTS[0]:
```
Comment (aj062) [inline Tools/CISupport/ews-build/steps.py:579]: This adds dependency on the order of elements in this list. If we need to do that, we should atleast add a comment where this list is defined indicating that the order matters.

**Review by aj062 — APPROVED** [review]

File: Tools/CISupport/ews-build/steps.py (hunk 1 above)
Comment (JonWBedard) [reply Tools/CISupport/ews-build/steps.py:579]: Not the first time we've done this, that being said, the comment is a good idea.

**Pull request merged**